src/actions.py

```diff
@@ -0,0 +1,32 @@
+import os
+from typing import Literal
+
+
+def env(name:str, _def=''):
+    """ Returns environment variable """
+    return os.environ.get(name, _def)
+
+
+#-bug: Previous step annotations are being overwritten

```

Comment:
This annotation related bug is still present.

Check [this issue](https://github.com/community/community/discussions/40606) too.

Reply:
This issue is not related to this PR. It will be fixed later.